src/utils/permissions.ts

```diff
@@ -239,6 +239,123 @@ export async function hasChannelPermissions(
   return hasPerms;
 }
 
+/** Checks if the bot has the permissions in a channel, if not error will thrown */
+export function botDependsChannelPermissions(
+  channelID: string,
+  permissions: Permission[],
+) {
+  return dependsChannelPermissions(channelID, botID, permissions);
+}
+
+/** Checks if a user has permissions in a channel, if not error will thrown */
+export async function dependsChannelPermissions(
```

Comment:
Why the name change from hasChannelPermissions to dependsChannelPermissions

Reply:
Thats not a name change, its a new function